keyboards/bandominedoni/config.h

```diff
@@ -158,3 +141,10 @@
 #endif  //  MIDI_ENABLE
 
 #define TAP_CODE_DELAY 10
+
+/*
+ * Encoder options
+ */
+#ifdef ENCODER_ENABLE
+#   define ENCODER_RESOLUTIONS_RIGHT { 4 }
+#endif  // ENCODER_ENABLE
```

Comment:
```suggestion
```

Reply:
I heard https://github.com/qmk/qmk_firmware/pull/20173#pullrequestreview-1347028091 that resolution = 4 is a default. However, when a keyboard is a split keyboard that has a rotary encoder only on the right-hand side, it seems "resolution = 4" nor the default value won't work.
Till  info.json supports the resolution setting on the right-hand side keyboard, this statement is needed.

Reply:
You wouldnt want to randomly introduce "legacy" config, when the rest is within `info.json`

```json
        "encoder": {
            "right": {
                "rotary": [
                    {"pin_a": "F5", "pin_b": "F4", "resolution": 4}
                ]
            }
        }
```

As the default is `4` this config is somewhat pointless. However it looks like #20179 should fix this scenario.

Reply:
I confirmed that https://github.com/qmk/qmk_firmware/pull/20179 will fix the problem, without the resolution setting, and without the statement below.
```
"encoder": {
        "rotary": []
    },
```
I'll delete them.